_components/button-group/accessibility-tests.md

```diff
@@ -0,0 +1,12 @@
+---
+permalink: /components/button-group/accessibility-tests/
+layout: accessibility-test
+component:
+ name: button group
+title: Button group accessibility tests
+category: Components
+lead: Any USWDS button group component should pass these manual accessibility tests.
+changelog:
+ key: 'component-button-group-accessibility'
+---
+{% include accessibility-tests/test-results-summary.html page='main' %}
```

Comment:
**issue:** Can you move this include to button-group.md, at the bottom of the file? We need this on the main component page.

```suggestion
```

Reply:
Updated in [3b54fef](https://github.com/uswds/uswds-site/pull/3009/commits/3b54fef685a10a1cfd5629082077abfd5369aa93)